tutorials/build_system/gradle_guide.md

```diff
@@ -0,0 +1,157 @@
+
+#### Running Against Alternate Versions of the JetBrains Runtime
+Every version of the IntelliJ Platform has a corresponding version of the [JetBrains Runtime](/basics/ide_development_instance.md#using-a-jetbrains-runtime-for-the-development-instance).
+A different version of the runtime can be used by specifying the `runIde.jbrVersion` attribute, describing a version of the JetBrains Runtime that should be used by the IDE Development Instance.
+The Gradle plugin will fetch the specified JetBrains Runtime as needed.
+
+### Managing Directories used by the Gradle Plugin
+There are several attributes to control where the Gradle plugin places directories for downloads and for use by the IDE Development Instance.
+
+The location of the [sandbox home](/basics/ide_development_instance.md#sandbox-home-location-for-gradle-based-plugin-projects) directory and its subdirectories can be controlled with Gradle plugin attributes.
+The `intellij.sandboxDirectory` attribute is used to set the path for the sandbox directory to be used while running the plugin in an IDE Development Instance. 
+Locations of the sandbox [subdirectories](/basics/ide_development_instance.md#development-instance-settings-caches-logs-and-plugins) can be controlled using the `runIde.configDirectory`, `runIde.pluginsDirectory`, and `runIde.systemDirectory` attributes.
+If the `intellij.sandboxDirectory` path is explicitly set, the subdirectory attributes default to the new sandbox directory.
+
+The storage location of downloaded IDE versions and components defaults to the Gradle cache directory.
+However, it can be controlled by setting the `intellij.ideaDependencyCachePath` attribute.
+
+### Controlling Downloads by the Gradle Plugin
+As mentioned in the section about [configuring the intellij platform](#configuring-the-gradle-plugin-for-building-intellij-platform-plugin-projects) used for building plugin projects, the Gradle plugin will fetch the version of the IntelliJ Platform specified by the default or by the `intellij` attributes.
+Standardizing the versions of the Gradle plugin and Gradle system across projects will minimize the time spent downloading versions.
+There are controls for managing the IntelliJ IDEA Gradle plugin version, and the version of Gradle itself.
+The Gradle plugin version is defined in the `plugins {}` section of a project's `build.gradle` file.
+The Gradle version is in defined in a project's `gradle-wrapper.properties`.
+
+### Patching the Plugin Configuration File
+A plugin project's `plugin.xml` file has values replaced from the attributes of the [Patching DSL](https://github.com/jetbrains/gradle-intellij-plugin#patching-dsl).
+With the exception of the `since-build` and `until-build` attributes of the `<idea-version>` element, all of the attributes in the Patching DSL will be substituted into the corresponding element values in a plugin project's `plugin.xml` file.
```

Comment:
why since-build and until-build are the exceptions? they also can be substituted using `patching dsl` and it's a preferable way doing it. So I'd omit describing `intellij.updateSinceUntilBuild` property

Reply:
In retrospect, I did not explain this correctly or well. Please see my understanding in the [issue comment](https://youtrack.jetbrains.com/issue/IJSDK-473#focus=streamItem-27-3517059.0-0).
If verbose comment is correct, I will try to summarize it in the document.